docs/user-guides/workspace-access/remote-desktops.md

```diff
@@ -88,32 +44,126 @@ locals {
+Connect using the [Amazon DCV Desktop client](https://docs.aws.amazon.com/dcv/latest/userguide/using-connecting.html) by forwarding the DCV port to your local machine:
 
-Our [Amazon DCV Windows](https://registry.coder.com/modules/amazon-dcv-windows)
-module adds a one-click button to open an Amazon DCV session in the browser.
-This requires just a few lines of Terraform in your template, see the
-documentation on our registry for setup.
+<div class="tabs">
 
-![Amazon DCV Windows Module in a Workspace](../../images/user-guides/amazon-dcv-windows-demo.png)
+#### Coder Desktop
+
+[Coder Desktop](../desktop/index.md)'s **Coder Connect** feature creates a connection to your workspaces in the background. Use DCV client to connect to `<workspace-name>.coder:8443`.
+
+#### CLI
+
+Use the following command to forward the DCV port to your local machine:
+
+```console
+coder port-forward <workspace-name> --tcp 8443:8443
+```
+
+</div>
+
+### Browser
+
+Our [Amazon DCV Windows](https://registry.coder.com/modules/amazon-dcv-windows) module adds a one-click button to open an Amazon DCV session in the browser. This requires just a few lines of Terraform in your template, see the documentation on our registry for setup.
+
+</div>
+
+![Amazon DCV](../../images/user-guides/amazon-dcv-windows-demo.png)
+
+## VNC
+
+The common way to connect to a desktop session of a Linux workspace is to use a VNC client. The VNC client can be installed on your local machine or accessed through a web browser. There is an additional requirement to install the VNC server on the workspace.
+
+Installation instructions vary depending on your workspace's operating system, platform, and build system. Refer to the [enterprise-desktop](https://github.com/coder/images/tree/main/images/desktop) image for a starting point.
+
+As a starting point, see the enterprise-desktop image. It can be used to provision a Dockerized workspace with the following software:
+
+- Ubuntu 24.04
+- XFCE Desktop
+- KasmVNC Server and Web Client
+
+<div class="tabs">
+
+### Desktop Client
+
+Use a VNC client (e.g., [TigerVNC](https://tigervnc.org/)) by forwarding the VNC port to your local machine.
+
+<div class="tab">
+
+#### Coder Desktop
+
+[Coder Desktop](../desktop/index.md)'s **Coder Connect** feature allows you to connect to your workspace's VNC server at `<workspace-name>.coder:5900`.
+
+#### CLI
+
+Use the following command to forward the VNC port to your local machine:
+
+```bash
+coder port-forward <workspace-name> --tcp 5900:5900
+```
+
+Now you can connect to your workspace's VNC server using a VNC client at `localhost:5900`.
+
+</div>
+
+### Browser
+
+The [KasmVNC module](https://registry.coder.com/modules/coder/kasmvnc) allows browser-based access to your workspace by installing and configuring the [KasmVNC](https://github.com/kasmtech/KasmVNC) server and web client.
+
+</div>
+
+![VNC Desktop in Coder](../../images/vnc-desktop.png)
+
+## Troubleshooting
+
+### RDP UDP Issues
```

Comment:
Not sure I agree with putting this at the end instead of in the RDP section. Seems less likely to get read.

Reply:
Moved the Note back up.